main.py

```diff
@@ -0,0 +1,53 @@
+from copy import deepcopy
+
+def primary_forbidden_sequences(list_of_all_forbidden_sequences,P):
+    all_sequences = deepcopy(list_of_all_forbidden_sequences)
+    index_to_be_taken = [0]*P
+    for i in range(P):
+        if index_to_be_taken[i] == 0:
+            current_sequence = all_sequences[i]
```

Comment:
It is more natural to write `for sequence in all_sequences:`

Reply:
If I do write `for sequence in all_sequences:` then I won't be able to access to my `index_of_all_sequence[i]` would I ?